dali_tf_plugin/make_dali_tf_sdist.sh

```diff
@@ -8,13 +8,28 @@ echo "Listing available *.so files:"
 find . -name '*.so' || true
 
 mkdir -p dali_tf_sdist_build
-cd dali_tf_sdist_build
+pushd dali_tf_sdist_build
+
+CUDA_VERSION_STR=$(echo $(ls /usr/local/cuda/lib64/libcudart.so*)  | sed 's/.*\.\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)/\1.\2/')
 
 cmake .. \
+      -DCUDA_VERSION:STRING="${CUDA_VERSION_STR}" \
       -DDALI_BUILD_FLAVOR=${NVIDIA_DALI_BUILD_FLAVOR} \
       -DTIMESTAMP=${DALI_TIMESTAMP} \
       -DGIT_SHA=${GIT_SHA}
-
 make -j install
 python setup.py sdist
 cp dist/*.tar.gz /dali_tf_sdist
+popd
+
+mkdir -p dummy/build
+pushd dummy/build
+cmake .. \
+      -DCUDA_VERSION:STRING="${CUDA_VERSION_STR}" \
+      -DDALI_BUILD_FLAVOR=${NVIDIA_DALI_BUILD_FLAVOR} \
+      -DTIMESTAMP=${DALI_TIMESTAMP} \
+      -DGIT_SHA=${GIT_SHA}
+python setup.py sdist
+
+mkdir -p /dali_tf_sdist/dummy
+cp dist/*.tar.gz /dali_tf_sdist/dummy
```

Comment:
Again parts of `make_nvidia_dali_tf_dummy.sh` duplicated. Do we need those `make_nvidia_dali*_dummy.sh` scripts?

Reply:
Also this one always builds the dummy package unconditionally. I guess we probably remove it at some point, right

Reply:
yes, we will remove it